### PR TITLE
UTOPIA-430: Questionaire Updates

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Updates PPQ Form to reflect iteration 2 of designs
- Re-configures interfaces to match questions
- Reworks styling to be more responsive and support different browsers
- Fixes weird Navbar margin issue that was occurring on some pages

### NOTE!!!
Form submission still responds with a 400 and I have yet to accurately diagnose the problem. Any help here would be greatly appreciated. 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [x] Version change

## How Has This Been Tested?

- Ran the app in multiple browsers to test compatibility
- Checked screen size responsiveness
- Submitted form to check API response (still WIP)

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [x] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/sass/_buttons.scss [hunk 1]
```diff
@@ -89,7 +89,8 @@ a:hover {
   }
 }
 
-.btn-login {
+.btn-login,
+%btn-login {
```
Comment (Bottle7) [inline src/frontend/src/sass/_buttons.scss:93]: I was receiving the same error that @binggaofw was getting when extending using '.btn-login' so simply added the placeholder version to be identically defined for use below.

File: src/frontend/src/utils/helper.util.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { check } from 'prettier';
+
```
Comment (Bottle7) [inline src/frontend/src/utils/helper.util.ts:2]: I did not add this import. No idea why GitHub thinks that I added this diff.

**Comment by binggaofw** [comment]
I think you did not merge latest main branch to your branch cause the 400 error. I checked the main branch, the ppq dto already update but your backend still have email etc.  after you merge with main,then
try run `docker system prune -a` to remove all the containers
then run  `docker volume rm $(docker volume ls -q)` to remove all the volumes
then run `make app-run` to start the app, it should be work

**Comment by Bottle7** [comment]
> I think you did not merge latest main branch to your branch cause the 400 error. I checked the main branch, the ppq dto already update but your backend still have email etc. after you merge with main,then try run `docker system prune -a` to remove all the containers then run `docker volume rm $(docker volume ls -q)` to remove all the volumes then run `make app-run` to start the app, it should be work

I followed your instructions and, unfortunately, I still receive a 400 response on POST.

File: src/frontend/src/constant/constant.ts [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
-    label: 'Information Sharing Agreement',
-    value: 'hasSharingOfPersonalInformation',
-    tooltip: true,
-    tooltipText: `An information sharing agreement allows a public sector organization, such as a ministry, 
-    to share personal information with another government or government agency on a regular basis. ISAs are 
-    not necessary when one B.C. government ministry shares personal information with another.`,
-  },
+  /*{
+    //label: 'Information Sharing Agreement',
+    //value: 'hasSharingOfPersonalInformation',
```
Comment (binggaofw) [inline src/frontend/src/constant/constant.ts:17]: i guess we can delete these unuse code or add a note mentioned we will use it in the future

**Review by binggaofw — APPROVED** [review]

**Pull request merged**